### PR TITLE
Ournetworks.ca First Phase Content + Design

### PR DESCRIPTION
**Pull request by dcwalk**
Okay, drawing on https://github.com/ournetworks/2018/issues/27 I've moved this into a PR for tweaking, I imagine this should be done in a manner that does not conflict with `redesign-pack-1`, I opted for a new branch as this is content and that is all sass/css cleanup rn

Things I imagine should happen:

- [x] add about page with organizers
- [x] Home page still has text, links to organizers? (didn't link b/c "About" is menu item)
- [x] fix sprint language
- [x] set up community billets sheet
- ~sort out fancy block for venue (includes? data?)~ Just added venue deets for now, not on page (2nd phase?)

### CODE REVIEW

**Comment by dcwalk** [comment]
Yo, also-- I love my Worksans Medium @garrying 😢

File: index.md [hunk 1]
```diff
@@ -1,60 +1,21 @@
...
-    {% include event.html event=call %}
-  </div>
-  {% endfor %}
-</section>
+The deadline for conference proposals has now closed, check back late-May for our full conference schedule! In the meantime, are you interested in [selling your stuff](/2018/kickoff-yami-ichi/) at our Internet Yami-ichi? Considering [hosting a project](/2018/sprints/) during our sprints?
 
 ### Updates
 
-Want to receive a update when registration launches?
+Want to receive a update mid-May when registration and the schedule launches?
```
Comment (dcwalk) [inline index.md]: an

File: 2018/sprints.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+| Project Name | Project Lead | Description and Goals | Attending |
+|:-------------|:-------------|:----------------------|:----------|
+|              |              |                       |           |
+
+
+## Propose a Sprint
+
+Anyone can propose a Sprint in the lead up to and at the conference. If a new idea comes up during Our Networks and you feel a Sprint may be the right format to test / kickstart the project, we encourage people pitch a Sprint.
+
+Pitching a Sprint requires a short description about the idea along with the following information:
```
Comment (dcwalk) [inline 2018/sprints.md]: need to make clear you PR this repo (and this page to add yr deets) Is there a PR template for that? (no right?)

File: 2018/sprints.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+We'll take care of providing space from 10am - 9:30pm Monday through Wednesday with hi-speed internet, power bars, post-its, and markers.
+
+## Join a Sprint
+
+Add your name to the list and make sure to watch the sprint pitches on Sunday at the conference, the project lead will be there providing an overview.
+
+| Project Name | Project Lead | Description and Goals | Attending |
+|:-------------|:-------------|:----------------------|:----------|
+|              |              |                       |           |
+
```
Comment (dcwalk) [inline 2018/sprints.md]: extra space

File: 2018/kickoff-yami-ichi.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Take your own Internet liberties* here, with us.
+
+* But no dangerous or illegal goods, please! This black market wants to remain
+free and useful!
+```
+
+Calling all vendors of internet-ish things! Come gather and exchange goods from the backstreets of the Internet Friday July 13, 2018 afternoon into evening.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/mjWJsE7B1cs" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (dcwalk) [inline 2018/kickoff-yami-ichi.md]: centre?

File: 2018/faq-qaf.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+We are committed to addressing barriers to access. So far we've confirmed a Conference and Sprint space that are entered by ramp and elevator, with accessible washrooms.
+
+### What is the deal with project sprints?
+
+Project sprints are a great way to work together to learn and create Do It With Others (DIWO)-style. The [Sprints](/2018/sprints/) page has a list of projects and details for proposing your own
+
+### I am traveling from out of town, how can I stay in Toronto affordably? (a.k.a. What are Community Billets?)
+
+We do not have a discount rate with a hotel, however we have arranged for a limited number of spots for those looking for a place to stay, check out the [community billets list](#) to coordinate!
```
Comment (dcwalk) [inline 2018/faq-qaf.md]: Need to think through this a little more -- would an open ethercalc spreadsheet work? I think we could have contact name, description of avail, intersection and then just the email part becomes ??? (set up billets@ournetworks.ca? have every billet person on it and they can respond as they should?)

File: 2018/conference.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+order: 2
+title: "2018"
+subtitle: "Conference"
+titleDisplay: "Conference"
+excerpt:
```
Comment (dcwalk) [inline 2018/conference.md]: maybe we strip some of this header info

File: 2018/conference.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+order: 2
+title: "2018"
```
Comment (dcwalk) [inline 2018/conference.md]: wondering if "section" is more appropriate?

File: 2018/conference.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
```
Comment (dcwalk) [inline 2018/conference.md:2]: Might want to refactor as I think the page | splash distinction is unhelpful rn

File: 2018/conference.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+order: 2
+title: "2018"
+subtitle: "Conference"
+titleDisplay: "Conference"
+excerpt:
+venueId:  
+date:
+startTime:
```
Comment (dcwalk) [inline 2018/conference.md]: Best way to capture timing details? venue? Imagining data for the schedule and program, potentially venues <-- but a venue is just for a single event (page) so maybe better handled in front matter

**Comment by dcwalk** [comment]
@patcon and @benhylau I need bios!

**Comment by dcwalk** [comment]
This is partially updated. @garrying and someone else, can I get a review tomorrow? 🙏

**Comment by dcwalk** [comment]
There are two links that are fake/failing:
- Community Billets sheet (??!? hosted ethercalc?)
- Sprints PR (https://github.com/ournetworks/ournetworks.ca/edit/master/2018/sprints.md) not on master rn!) (edit: added `data-proofer-ignore` for now)

File: yami-ichi.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+titleDisplay: "Yami-Ichi"
+excerpt:
+startDate: 2018-07-13
+redirect_from:
+  - '/2018/kickoff-yami-ichi/'
+---
+
+Friday • 13
+
+```
```
Comment (garrying) [inline yami-ichi.md]: Maybe this could be block quote instead with reference?

File: yami-ichi.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+title: "2018"
+subtitle: "Kickoff Yami-ichi"
+titleDisplay: "Yami-Ichi"
+excerpt:
+startDate: 2018-07-13
+redirect_from:
+  - '/2018/kickoff-yami-ichi/'
+---
+
+Friday • 13
```
Comment (garrying) [inline yami-ichi.md]: Could this be `Friday • July 13`?
This would get carried into `/conference/` and `/sprints/`.

File: 2018/about.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+Our Networks is a conference about the past, present, and future of building our own network infrastructures. Now in its second year, the event brings together enthusiasts, hardware and software hackers, researchers, organizers and more to collectively explore creative and critical engagements with the Internet and alternative infrastructures.
+
+### I want to make this even even better, how can I help?
+
+Awesome, we are [planning this event openly](https://github.com/ournetworks/2018)! You should consider [tabling at our Yami-ichi](/yami-ichi/) or [pitching a project for our Sprints](/2018/sprints/). If those activities don't appeal, we will need volunteers to help the event run smoothly--reach out to us at orga@ournetworks.ca!
+
+###  Organizing Team
+
+Our Networks organizers have planned workshops and hosted events on network literacy and community networks as well as built mesh networks with Toronto Mesh for the past two years:
```
Comment (garrying) [inline 2018/about.md]: Thinking a comma somewhere in this sentence.

File: 2018/sprints.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+| Project Name            | Project Lead | Description and Goals | Attending |
+|:------------------------|:-------------|:----------------------|:----------|
+|                         |              |                       |           |
+
+<br />
+
+## Propose a Sprint
+
+Anyone can propose a Sprint in the lead up to--and at--the conference. If a topic comes up during Our Networks and you feel a Sprint may be the right format to test / kickstart the project, we encourage you to propose a Sprint!
```
Comment (garrying) [inline 2018/sprints.md]: `test / kickstart` -> `test/kickstart`

File: yami-ichi.md (hunk 10 above)
Comment (dcwalk) [reply yami-ichi.md]: Sure! Tho I think the amt of times July is on the homepage is a bit distracting -- maybe we swap?

File: _includes/2018-hero.html [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+  <header>
+    <h2 style="margin-bottom: 0;">July 13–18, 2018</h2>
+    <h1 style="margin-bottom: 0;">{{ site.title }}</h1>
+    <h3 style="margin-bottom: 1.5em; margin-top: 0;">Toronto, Ontario</h3>
+  </header>
+  <!-- Event hero section -->
+  <section>
+    <div class="feature">
+      <h3></h3>
+      <h4 class="splash-heading">Kickoff Yami-ichi {% include graphics/bullet.svg %} July 13</h4>
```
Comment (dcwalk) [inline _includes/2018-hero.html]: Can we remove "July" here?

File: _includes/2018-hero.html [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+      <p>Network field trips and Toronto's first internet yami-ichi</p>
+    </div>
+    <div class="feature">
+      <h3></h3>
+      <h4 class="splash-heading">Conference {% include graphics/bullet.svg %} July 14–15</h4>
```
Comment (dcwalk) [inline _includes/2018-hero.html]: Can we remove "July" here?

File: _includes/2018-hero.html [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+      <p>Our main event, a weekend full of talks, demos, and workshops</p>
+    </div>
+    <div class="feature">
+      <h3></h3>
+      <h4 class="splash-heading">Sprints {% include graphics/bullet.svg %} July 16–18</h4>
```
Comment (dcwalk) [inline _includes/2018-hero.html]: Can we remove "July" here?

File: _includes/2018-hero.html (hunk 15 above)
Comment (dcwalk) [reply _includes/2018-hero.html]: Also, could we follow the format we established of "14, 15"

File: yami-ichi.md (hunk 11 above)
Comment (dcwalk) [reply yami-ichi.md]: changed

Comment (dcwalk) [reply yami-ichi.md]: Sure! Tho I think the amt of times July is on the homepage is a bit distracting -- maybe we swap?

File: 2018/about.md (hunk 12 above)
Comment (benhylau) [reply 2018/about.md]: Suggestion:

>Our Networks organizers have facilitated workshops, hosted network literacy events, and built community networks with Toronto Mesh for the past two years:

File: yami-ichi.md (hunk 10 above)
Comment (dcwalk) [reply yami-ichi.md]: changed

File: yami-ichi.md (hunk 11 above)
Comment (dcwalk) [reply yami-ichi.md]: changed

File: index.md [hunk 17]
```diff
@@ -1,70 +1,32 @@
...
 
-Our theme this year focuses on how we **Do It With Others (DIWO)** as opposed to just ourselves, building on the "**distributed campaign for emancipatory, networked art practices**," instigated by UK-based [Furtherfield](http://archive.furtherfield.org/projects/diwo-do-it-others-resource) (2006) as a response to the Do It Yourself (DIY) movement. We have seen a growing number of peer-to-peer, inclusive, and privacy-respecting projects mobilizing in favour of resilient, accessible, equitable communications over the internet in 2017. And yet many open questions remain. In the face of threats to the open internet, which tools and tactics will help us recognize the opportunities and challenges of this moment? What kinds of creative and critical engagement with technology practices can enable meaningful change when we do it with others?
+The deadline for conference proposals has now closed, check back late-May for our full conference schedule! In the meantime, are you interested in [selling your stuff](/2018/kickoff-yami-ichi/) at our Internet Yami-ichi? Considering [hosting a project](/2018/sprints/) during our sprints?
```
Comment (benhylau) [inline index.md]: `s/Considering/Consider`? Otherwise it would imply `"interested in" considering`

Comment (benhylau) [reply index.md]: `Sprint` is capitalized in some places. I think they should all be capitalized.

**Review by benhylau — DISMISSED** [review]

File: 2018/about.md (hunk 12 above)
Comment (dcwalk) [reply 2018/about.md]: restructured

**Comment by benhylau** [comment]
Added some minor comments to the text. Cannot test the site because my env is beyond broke today.

File: index.md (hunk 17 above)
Comment (dcwalk) [reply index.md]: Thanks @benhylau -- revised sentence and fixed a couple lower case sprints

File: yami-ichi.md (hunk 11 above)
Comment (garrying) [reply yami-ichi.md]: @dcwalk Swap makes sense. Taking another look and thinking these could be a headings, or pulled closer to the post title?

File: 2018/conference.md (hunk 8 above)
Comment (dcwalk) [reply 2018/conference.md:2]: actually valuable! this should stay :)

**Comment by dcwalk** [comment]
Okay @garrying -- this is in your court, don't look too closely at my janky liquid tags in https://github.com/ournetworks/ournetworks.ca/pull/19/commits/ff20f59a7a544444772d5991b1a7213f3ae648fc unless you want some cringe action.

File: 2018/about.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+
+**Garry Ing**{:#garry} is a designer and technologist residing in Toronto, contributing to Toronto Mesh. His previous work and collaborations has been with the Strategic Innovation Lab (sLab) at OCAD University, the Technologies for Aging Gracefully Lab at the University of Toronto, and Normative. He is a graduate of OCAD University, with a background in graphic design.
+
+<ul class="bio-sm-list">
+  <li class="bio-sm-list-item"><a href="https://garrying.com/" target="_blank" data-proofer-ignore>{% include icons/link.svg %}&nbsp;garrying.com</a></li>
+  <li class="bio-sm-list-item"><a href="https://twitter.com/garrying" target="_blank">{% include icons/twitter.svg %}&nbsp;@garrying</a></li>
+  <li class="bio-sm-list-item"><a href="https://github.com/garrying" target="_blank">{% include icons/github.svg %}&nbsp;garrying</a></li>
+</ul>
+
+**Patrick Connolly**{:#patcon} is a failed biochemist, a computer programmer, a co-organizer of CivicTech Toronto, and an anarchist sympathizer. He is interested in liquid democracy, the changing nature of work, and how we can build communities and organizations that are more open, inclusive, and resilient.
```
Comment (patcon) [inline 2018/about.md:46]: :+1: :raised_hands:

**Pull request merged**